project/src/main/java/com/google/autograder/data/Database.java

```diff
@@ -57,7 +57,8 @@ public static void delete(Entity entity) {
         return DATA_STORE.prepare(query).asIterable();
     }
 
-    // Takes in a courses array in JSON format and then stores that courses data for the current user.
+    /** Takes in a courses array in String JSON format and then stores that courses data for the current user.
```

Comment:
The format of javadoc is something like:
```
/**
 * Stores course data for the current user.
 *
 * @param coursesJSON the courses in JSON form
 */
```
The first part is the summary and should describe what the method does. Afterwards you can describe with `@param`, `@return`, `@throw` more details about the method.

Reply:
thanks. implemented